apps/astarte_realm_management/lib/astarte_realm_management/queries.ex

```diff
@@ -579,14 +579,13 @@ defmodule Astarte.RealmManagement.Queries do
 
     consistency = Consistency.device_info(:read)
 
-    properties =
-      query
-      |> Repo.fetch_all(prefix: keyspace, consistency: consistency)
-      |> Enum.map(fn property ->
+    with {:ok, properties} <- Repo.fetch_all(query, prefix: keyspace, consistency: consistency) do
+      Enum.map(properties, fn property ->
         [endpoint_id: property.endpoint_id, path: property.path]
       end)
 
-    {:ok, properties}
+      {:ok, properties}
```

Comment:
The `Enum.map` line seems to produce a value that is not used: you forgot to rebind `properties` to the newly computed value